build.gradle.kts

```
@@ -5,6 +5,18 @@ plugins {
     alias(libs.plugins.kotlinxSerialization).apply(false)
     alias(libs.plugins.kotlinCocoapods).apply(false)
     alias(libs.plugins.swift.klib).apply(false)
+    id("io.github.gradle-nexus.publish-plugin") version "2.0.0"
     id("com.netflix.nebula.release") version "19.0.8"
     id("org.jetbrains.kotlinx.binary-compatibility-validator") version "0.16.3"
+}
+
+group = "com.doordeck" // Required by nexusPublishing
```

```suggestion
group = "com.doordeck"
```

It's probably sensible to have anyway